src/cms/templates/pages/page_revisions.html

```diff
@@ -5,21 +5,33 @@
 {% load widget_tweaks %}
 {% load rules %}
 
+<div class="w-full p-4 flex justify-end">
+<a href="{% url 'edit_page' page_id=page.id region_slug=region.slug language_slug=language.slug %}" class="bg-gray-400 hover:bg-gray-500 cursor-pointer text-white font-bold py-3 px-4 rounded">
+    {% trans 'Go Back to Page Editor' %}
+</a>
+</div>
+
 <div class="flex flex-wrap mb-4">
     <div class="w-full flex flex-wrap flex-col justify-center">
         <h1 class="heading">
             {% with page_translations.first.title as page_title %}
-            {% blocktrans %}Page revisions of "{{ page_title }}"{% endblocktrans %}
+            {% blocktrans %}Page versions of "{{ page_title }}"{% endblocktrans %}
             {% endwith %}
         </h1>
     </div>
 </div>
 
 <form method="post" action="{% url 'page_revisions' page_id=page.id region_slug=region.slug language_slug=language.slug %}">
     {% csrf_token %}
-    <div class="w-full mb-20 relative">
+    <div class="w-3/5 m-auto mb-20 relative">
```

Comment:
```suggestion
    <div class="w-3/5 m-auto mb-28 relative">
```
Since the editor adds one additional line, I suggest to increase the margin here.